package.json

```diff
@@ -1,13 +1,15 @@
 {
   "type": "module",
   "devDependencies": {
+    "amqplib": "^0.10.3",
     "cors": "^2.8.5",
     "express": "^4.18.2",
     "mysql2": "^3.9.1",
     "nodemon": "^3.0.3",
     "sequelize": "^6.35.2"
   },
   "dependencies": {
+    "body-parser": "^1.20.2",
```

Comment:
We do not need this. Remove this package

Reply:
Done